tests/fixtures/markdown/export/expected.md

```diff
@@ -0,0 +1,66 @@
+# urn:datacontract:checkout:orders-latest
+## Info
+*Successful customer orders in the webshop. <br>All orders since 2020-01-01. <br>Orders with their line items are in their current state (no history included).<br>*<br>• **title:** Orders Latest<br>• **version:** 2.0.0<br>• **owner:** Checkout Team<br>• **contact:** {'name': 'John Doe (Data Product Owner)', 'url': 'https://teams.microsoft.com/l/channel/example/checkout'}
+
+## Servers
+| Name | Type | Attributes |
+| ---- | ---- | ---------- |
+| production | s3 | *One folder per model. One file per day.*<br>• **environment:** prod<br>• **format:** json<br>• **delimiter:** new_line<br>• **location:** s3://datacontract-example-orders-latest/v2/{model}/*.json<br>• **roles:** [{'name': 'analyst_us', 'description': 'Access to the data for US region'}, {'name': 'analyst_cn', 'description': 'Access to the data for China region'}] |
+
+## Terms
+*No description.*<br>• **usage:** Data can be used for reports, analytics and machine learning use cases.
```

Comment:
we should use standard markdown here as well

Reply:
fixed